backend-project/config/urls.py

```diff
@@ -56,6 +56,7 @@
         name="schema_swagger",
     ),
     path("api/", include(router.urls)),
+    path("api/auth/", include("rest_framework_social_oauth2.urls")),
```

Comment:
Czy jesteś zadowolony z sposobu udokumentowania API dla uwierzytelniania?

Reply:
Nie jestem zadowolony. Brakuje udokumentowanego endpointu /api/auth/login/google-oauth2/. Reszta endpointów nie posiada opisanych parametrów.
Jak uda mi się poprawić kod by testy przechodziły to wezmę się za udokumentowanie endpointów.